src/serlio/materials/MaterialUtils.cpp

```diff
@@ -22,31 +22,45 @@ constexpr const wchar_t* RGBA8_FORMAT = L"RGBA8";
 constexpr const wchar_t* FORMAT_STRING = L"format";
 
 adsk::Data::Structure* createNewMaterialInfoMapStructure() {
-	adsk::Data::Structure* fStructure;
-
 	// Register our structure since it is not registered yet.
-	fStructure = adsk::Data::Structure::create();
+	adsk::Data::Structure* fStructure = adsk::Data::Structure::create();
 	fStructure->setName(PRT_MATERIALINFO_MAP_STRUCTURE.c_str());
 
 	fStructure->addMember(adsk::Data::Member::eDataType::kUInt64, 1, PRT_MATERIALINFO_MAP_KEY.c_str());
-	fStructure->addMember(adsk::Data::Member::eDataType::kString, 1, PRT_MATERIALINFO_MAP_VALUE.c_str());
+	fStructure->addMember(adsk::Data::Member::eDataType::kUInt8, 16, PRT_MATERIALINFO_MAP_VALUE.c_str());
```

Comment:
should avoid hardcoding '16', maybe use a constant and assert that Muuid.asString().length() == 16

Reply:
Actually, .asString() should be 36 characters long. But the Maya API provides an alternative storage solution (either as four uint32's or 16 uint8. This storage is more dense and therefore uses less characters. [see MUuid API](https://help.autodesk.com/view/MAYAUL/2022/ENU/?guid=Maya_SDK_cpp_ref_class_m_uuid_html).
This is why I was comfortable "hardcoding" the length, but It would still make sense to use a descriptive constant for it.

Reply:
Using constant now.